spec/javascripts/health-care/components/form-elements/ErrorableTextInput.spec.jsx

```diff
@@ -17,47 +18,47 @@ describe('<ErrorableTextInput>', () => {
 
     it('label is required', () => {
       SkinDeep.shallowRender(
-        <ErrorableTextInput onValueChange={(_update) => {}}/>);
+        <ErrorableTextInput field={makeField(1)} onValueChange={(_update) => {}}/>);
```

Comment:
Since we have a different component for number inputs, perhaps we should pass this a string instead of a number?